sled-diagnostics/src/queries.rs

```diff
@@ -1,18 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Wrapper for command execution with timeout.
+
 use std::{process::Command, time::Duration};
 
-use futures::{stream::FuturesUnordered, StreamExt};
-use illumos_utils::{dladm::DLADM, zone::IPADM, PFEXEC, ZONEADM};
 use thiserror::Error;
 use tokio::io::AsyncReadExt;
 
-const DEFAULT_TIMEOUT: Duration = Duration::from_secs(10);
+const DLADM: &str = "/usr/sbin/dladm";
+const IPADM: &str = "/usr/sbin/ipadm";
+const PFEXEC: &str = "/usr/bin/pfexec";
+const ZONEADM: &str = "/usr/sbin/zoneadm";
+
+pub const DEFAULT_TIMEOUT: Duration = Duration::from_secs(10);
 
-pub trait SupportBundleCommandHttpOutput {
+pub trait SledDiagnosticsCommandHttpOutput {
     fn get_output(self) -> String;
 }
 
```

Comment:
I know this isn't new in this PR, but since we're exposing it from a crate API, we may want to revisit this trait name.

- The name is a little redundant with the crate
- The `HttpOutput` bit is especially confusing now, since this trait also has nothing to do with HTTP

Reply:
I left this as is because I knew it would be addressed in a PR on top of this PR.
The updated output type can be found in #7228